deployment/distribution/build.gradle

```diff
@@ -24,8 +25,14 @@ applicationDistribution.from('src/main/resources/') {
     into 'conf'
 }
 
-applicationDistribution.from(project(':plugins:ddf-plugins').file('build/libs')) {
-    into 'plugins'
+project(':external:implementations').getSubprojects().each{ subProject ->
+    System.out.println(subProject.getName())
```

Comment:
❓ Do we want to keep this print statement here or was this just for debugging during development?

Reply:
It was for debugging. Removing.